tutoraspects/plugin.py

```diff
@@ -50,6 +50,8 @@
             ],
         ),
         ("EVENT_SINK_CLICKHOUSE_MODELS", ["course_overviews", "user_profile"]),
+        ("EVENT_SINK_CLICKHOUSE_PII_MODELS", ["user_profile", "external_id"]),
+        ("ASPECTS_ENABLE_PII", False),
```

Comment:
Can you put a comment here like: "Turning on this flag will store personally identifiable information in the ClickHouse database. Make sure that you understand the legal consequences of data storage and privacy before turning this on!